completion/README.md

```diff
@@ -95,4 +95,16 @@ exec $SHELL -l
 
 ## Fish
 
-TODO (#315): installation instructions
+### Mac (via Homebrew)
+Please look at the section about [installation via Homebrew](../README.md#using-homebrew-macos)
+``brew install git-machete`` automatically installs fish completion files for ``git machete``.
+
+### Linux
+
+Place the completion script in `/path/to/fish/completions/` (typically `~/.config/fish/completions/git-machete.fish`). You may have to create this directory if it doesn't exist.
+
+```shell script
+mkdir -p ~/.config/fish/completions
+curl -L https://raw.githubusercontent.com/VirtusLab/git-machete/develop/completion/git-machete.fish -o ~/.config/fish/completions/git-machete.fish
+echo "source ~/.config/fish/completions/git-machete.fish >/dev/null" >> ~/.config/fish/config.fish
```

Comment:
Hmmm I've actually tried that on my Fedora... ended up with the following:
```
➜ 13:14 ~/phoenix-backend chore/GMV3-2305-akka-http-bump PR #1410 $ fish
complete: Expected argument for option -C
~/.config/fish/completions/git-machete.fish (line 7): 
complete -C "git "
^
from sourcing file ~/.config/fish/completions/git-machete.fish
	called on line 1 of file ~/.config/fish/config.fish

from sourcing file ~/.config/fish/config.fish
	called during startup


       complete

   Synopsis
       complete ( -c | --command | -p | --path ) COMMAND
               [( -c | --command | -p | --path ) COMMAND]...
               [( -e | --erase )]
               [( -s | --short-option ) SHORT_OPTION]...
               [( -l | --long-option | -o | --old-option ) LONG_OPTION]...
               [( -a | --arguments ) OPTION_ARGUMENTS]
               [( -k | --keep-order )]
               [( -f | --no-files )]
               [( -r | --require-parameter )]
               [( -x | --exclusive )]
               [( -w | --wraps ) WRAPPED_COMMAND]...
               [( -n | --condition ) CONDITION]
               [( -d | --description ) DESCRIPTION]
       complete ( -C[STRING] | --do-complete[=STRING] )

complete: Type “help complete” for related documentation

Welcome to fish, the friendly interactive shell
plipski@localhost ~/phoenix-backend> 
```

Reply:
Pls investigate 🕵🏻

Reply:
Btw, despite the error, the completion for `git machete` and `git` itself works as expected

Pls also test for the corner case of completion for `git-machete`

Reply:
Since the older versions `complete -C "git "` is not allowed, to keep compatibility let's use `complete --do-complete="git "` instead.

Reply:
Okay! working now, resolving the thread